guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,38 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
+
+{Team} recommends performing operating system updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
+
+If there are any released {ProjectServer} updates, those will be included when performing this update.
+We do not recommend updating the operating system independently of {ProjectServer}.
```

Comment:
Oh, this is tricky. I gave it a try based on your conversation here but, well, I don't know:

```suggestion
// Starting with an introduction that focuses on what the user action is and what benefits it brings:
Update your {ProjectServer} and its underlying operating system regularly.
Minor releases patch security vulnerabilities and issues discovered since the previous versions.

// Clarifying the additional information and what to avoid.
[IMPORTANT]
====
Always update the operating system together with updating your {ProjectServer}.
Updating the operating system includes all released {ProjectServer} updates.
====
```

Among other things, I suggest to switch from "do not update independently" to "always update together with" (if that's what you meant). A positive recommendation takes less effort on the user's side to understand than a negative one.